src/PowerShellEditorServices.Protocol/Server/LanguageServer.cs

```diff
@@ -236,14 +236,32 @@ protected Task Stop()
             RequestContext<object> requestContext)
         {
             if (helpParams == null) { helpParams = "get-help"; }
-
-            var psCommand = new PSCommand();
-            psCommand.AddCommand("Get-Help");
-            psCommand.AddArgument(helpParams);
-            psCommand.AddParameter("Online");
-
-            await editorSession.PowerShellContext.ExecuteCommand<object>(psCommand);
-
+            string script = @"
+function __Check-Help {
+    param (
+        $Command
```

Comment:
Should be `[string] $Command`.  Also would prefer the name `CommandName`, since you won't be passing it a command object.